src/Keyboard.js

```diff
@@ -18,7 +18,10 @@ export default class Keyboard extends PureComponent {
 		isFirstLetterUppercase: PropTypes.bool,
 		layouts: PropTypes.arrayOf(PropTypes.shape({
 			symbolsKeyValue: PropTypes.string,
-			layout: PropTypes.arrayOf(PropTypes.string),
+      layout: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
+        PropTypes.arrayOf(PropTypes.string),
```

Comment:
we don't need this type, only `PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string))` is valid